lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3065,7 +3098,12 @@ namespace Js
 
         // Rundown on all existing functions and change their thunks so that they will go to debug mode once they are called.
 
+#if !defined(_WIN32) && ENABLE_NATIVE_CODEGEN
+        OldCodeGenAutoDelete autoDelete(this);
+        HRESULT hr = OnDebuggerAttachedDetached(/*attach*/ true, &(autoDelete.oldCodegen));
+#else // !defined(_WIN32) && ENABLE_NATIVE_CODEGEN
         HRESULT hr = OnDebuggerAttachedDetached(/*attach*/ true);
+#endif
```

Comment:
Is it necessary to have it differently on windows and xpalt? seems like xpalt way is valid for windows as well.

Reply:
Indeed no obvious reason why it shouldn't work. Still didn't want to change it on Windows as I'm not comfortable with Debugger related implementation.Thus, not able to estimate there well.

If you say it's good to go, I can remove ifdefs and enable it for Windows too.

Reply:
Remove the ifdefs and trigger a cirro crawler run. good to have same code on all platform as far as possible.